catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -332,6 +332,7 @@ private Filter forIds(List<String> ids) {
                 .filter(ci -> ci.getQualifier() == null || ci.getQualifier()
                         .equals(""))
                 .map(ContentItem::getMetacard)
+                .filter(Objects::nonNull)
```

Comment:
Since I refactored this method in my PR to fix this, I went ahead and added this line to where I refactored it to. So if you wish you can drop this to avoid conflicts.